rest-dashboard_integration_test.go

```diff
@@ -39,6 +39,10 @@ func Test_Dashboard_CRUD(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	if boardLinks, err = client.SearchDashboardsInFolders(ctx, []uint{1, 2}, "", false); err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Could we add a check here and before this on the length of `boardLinks`? It was unclear before but now it is even less clear if there are any elements in `boardLinks`. Because if there aren't then we won't actually check anything in the following loop. I think that's why the coverage has dropped. WDYT?

Reply:
Thanks for the response @GiedriusS!

Yes, you are right, the test was a bit obscure and I make it more indeed,

I refactored a bit and used the `0` ID (General) folder so most of the time return something.